lib/KImg.vue

```diff
@@ -14,6 +14,8 @@
 
 <script>
 
+  import kolibriLogo from '../docs/common/DocsPageTemplate/SideNav/kolibri-logo.svg';
```

Comment:
I'm not sure if we should be linking to an SVG in the docs directory here. Perhaps it'd be worth copy-pasting it directly into the same directory as this file?

Reply:
Definitely would be good to copy it to `KImg`/`KLogo` directory

Reply:
I doubt that someone re-organizing docs code and pages would get an idea to check whether it broke something in library. So good to have strict separation.